src/interfaces/node.h

```diff
@@ -6,6 +6,7 @@
 #define BITCOIN_INTERFACES_NODE_H
 
 #include <consensus/amount.h>          // For CAmount
+#include <logging.h>                   // For BCLog
```

Comment:
I think "For ..." comments are discouraged from use in new code because of maintainability